gooddata-java/src/main/java/com/gooddata/sdk/service/hierarchicalconfig/HierarchicalConfigService.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2004-2021, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.service.hierarchicalconfig;
+
+import com.gooddata.sdk.common.GoodDataException;
+import com.gooddata.sdk.model.hierarchicalconfig.SettingItem;
+import com.gooddata.sdk.model.hierarchicalconfig.Settings;
+import com.gooddata.sdk.model.project.Project;
+import com.gooddata.sdk.service.AbstractService;
+import com.gooddata.sdk.service.GoodDataSettings;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import static com.gooddata.sdk.common.util.Validate.notEmpty;
+import static com.gooddata.sdk.common.util.Validate.notNull;
+import static com.gooddata.sdk.model.hierarchicalconfig.SettingItem.PROJECT_SETTING_TEMPLATE;
+import static com.gooddata.sdk.model.hierarchicalconfig.Settings.PROJECT_SETTINGS_TEMPLATE;
+import static com.gooddata.sdk.service.hierarchicalconfig.HierarchicalConfigValidator.isItProjectSetting;
+
+/**
+ * Provides hierarchical configuration management. For more detailed description, see this documentation
+ * https://help.gooddata.com/api#/reference/hierarchical-configuration
+ */
+public class HierarchicalConfigService extends AbstractService {
+
+    public HierarchicalConfigService(RestTemplate restTemplate, GoodDataSettings settings) {
+        super(restTemplate, settings);
+    }
+
+    /**
+     * Returns aggregated project settings for given project.
+     *
+     * @param project project, cannot be null
+     * @return settings for given project
+     */
+    public Settings projectSettings(final Project project) {
```

Comment:
see naming in https://github.com/gooddata/gooddata-java/blob/master/CONTRIBUTING.md#services

Reply:
Fixed